client/reducers/index.js

```diff
@@ -42,6 +44,10 @@ export const getStory = (state: Object, storyUid: string) => state.stories[story
 export const getStories = (state: Object) => state.stories
 export const getStoriesList = (state: Object) => listFromMapping(state.stories)
 
+export const getGroup = (state: Object, groupUid: string) => state.groups(groupUid)
```

Comment:
We're not using this so for now it can probably just be removed, but regardless those are supposed to be square brackets, not round ones:
`state.groups[groupUid]` 😄